library/Requests/Capability.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Capability interface declaring the known capabilities
+ *
+ * @package Requests
+ * @subpackage Capability
+ */
+
+/**
+ * Capability interface declaring the known capabilities
+ *
+ * This is used as discovery to figure out which capabilities can be queried.
+ *
+ * @package Requests
+ * @subpackage Capability
+ */
+class Requests_Capability {
+
+	/**
+	 * Support for SSL.
+	 *
+	 * @var string
+	 */
+	const SSL = 'ssl';
+
+	/**
+	 * Collection of all known capabilities.
```

Comment:
Looks like this constant isn't used. Do we want to add dead code ?

```suggestion
	 * Collection of all capabilities supported in Requests.
	 *
	 * Note: this does not automatically mean that the capability will be supported for your chosen transport!
```